src/ui/handler/scroll_zoom.js

```diff
@@ -60,6 +60,9 @@ class ScrollZoomHandler {
     _defaultZoomRate: number;
     _wheelZoomRate: number;
 
+    _requireCtrl: boolean;
+    _container: HTMLElement;
```

Comment:
I feel like containers are not all that uncommon as a dummy element just to pick up mouse events, so maybe a friendly name to help indicate its purpose?

```suggestion
    _messageContainer: HTMLElement;
```